src/main/java/se/sundsvall/smloader/service/SupportManagementService.java

```diff
@@ -0,0 +1,76 @@
+
+import generated.se.sundsvall.supportmanagement.Errand;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import se.sundsvall.smloader.integration.db.CaseMappingRepository;
+import se.sundsvall.smloader.integration.db.CaseRepository;
+import se.sundsvall.smloader.integration.db.model.DeliveryStatus;
+import se.sundsvall.smloader.integration.supportmanagement.SupportManagementClient;
+import se.sundsvall.smloader.service.mapper.OpenEMapper;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
+
+import static java.util.stream.Collectors.toMap;
+import static org.springframework.http.HttpHeaders.LOCATION;
+import static se.sundsvall.smloader.integration.db.model.DeliveryStatus.CREATED;
+import static se.sundsvall.smloader.integration.db.model.DeliveryStatus.FAILED;
+import static se.sundsvall.smloader.integration.util.ErrandConstants.MUNICIPALITY_ID;
+import static se.sundsvall.smloader.integration.util.ErrandConstants.NAMESPACE_BY_FAMILY_ID;
+import static se.sundsvall.smloader.service.mapper.CaseMapper.toCaseMapping;
+
+@Service
+public class SupportManagementService {
+
+	private static final Logger LOGGER = LoggerFactory.getLogger(SupportManagementService.class);
+	private final SupportManagementClient supportManagementClient;
+	private final CaseRepository caseRepository;
+	private final CaseMappingRepository caseMappingRepository;
+	private final Map<String, OpenEMapper> openEMapperMap;
+
+	public SupportManagementService(SupportManagementClient supportManagementClient, CaseRepository caseRepository, CaseMappingRepository caseMappingRepository, final List<OpenEMapper> openEMappers) {
+		this.supportManagementClient = supportManagementClient;
+		this.caseRepository = caseRepository;
+		this.caseMappingRepository = caseMappingRepository;
+		this.openEMapperMap = openEMappers.stream().collect(toMap(OpenEMapper::getSupportedFamilyId, Function.identity()));
+	}
+
+	public void exportCases() {
+		final var casesToExport = caseRepository.findAllByDeliveryStatus(DeliveryStatus.PENDING);
+
+		casesToExport.forEach(caseEntity -> {
+			final var mapper = openEMapperMap.get(caseEntity.getFamilyId());
+			if (mapper == null) {
+				caseRepository.save(caseEntity.withDeliveryStatus(FAILED));
+				LOGGER.error("No mapper found for familyId: {}", caseEntity.getFamilyId());
+				return;
+			}
+
+			final var errand = mapper.mapToErrand(Optional.ofNullable(caseEntity.getOpenECase()).orElse("").getBytes());
+			final var errandId = sendToSupportManagement(errand, caseEntity.getFamilyId());
+			if (errandId == null) {
+				caseRepository.save(caseEntity.withDeliveryStatus(FAILED));
+				return;
+			}
+			final var caseMapping = toCaseMapping(errandId, caseEntity);
+			caseMappingRepository.save(caseMapping);
+			caseRepository.save(caseEntity.withDeliveryStatus(CREATED));
+			//TODO: Remove case from caseRepository?
```

Comment:
When will this TODO be solved?

Reply:
It will be more changes in service before it's ready. I'm still waiting for some requirements. The goal for this pr is to be able testing towards OpenE.